mycroft/util/lang/parse_en.py

```diff
@@ -927,6 +927,11 @@ def date_found():
                     hasYear = True
                 else:
                     hasYear = False
+
+            # if no date indicators found, it may not be the month of May
+            elif word == 'may' and wordNext == 'i':
```

Comment:
Should we make this a list so this becomes

```
elif word == 'may' and wordNext in ['i', 'we', 'be']
```

covering *may we...*, *... as it may be* as well as the i case.